routes/orderRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+
+const orderController = require("../controllers/orderController.js");
+const { loginRequired } = require("../middlewares/auth.js");
+const router = express.Router();
+
+router.get("/carts", loginRequired, orderController.getOrderPageForCartItems);
+router.post("/carts/result", loginRequired, orderController.orderResult);
```

Comment:
RESTful API 엔드포인트를 설계할 때 이미 index.js 파일에서 carts라는 리소스를 작성해주었습니다.

이미 postman을 통해 테스트를 해보셨기 때문에 엔드포인트에 `/carts/carts`와 같이 같은 리소스가 두 번 반복되는 것을 눈치채셨을 텐데요,

그렇기 때문에 해당 파일에서 carts라는 리소스를 다시 작성해줄 필요가 없습니다.
```suggestion
router.get("/", loginRequired, orderController.getOrderPageForCartItems);
router.post("/", loginRequired, orderController.orderResult);
```

Reply:
저 코드에서의 엔드포인트는 orders여서 carts 라는 표시가 필요하다고 생각했습니다..